mentat/terminal/output.py

```diff
@@ -29,13 +31,22 @@ def print_stream_message(message: StreamMessage):
     end = "\n"
     color = None
     flush = False
+    session_context = SESSION_CONTEXT.get()
+    if session_context.config.theme is not None:
+        theme = Themes.get(session_context.config.theme)
+    else:
+        theme = None
     if message.extra:
         if isinstance(message.extra.get("end"), str):
             end = message.extra["end"]
         if isinstance(message.extra.get("color"), str):
             color = message.extra["color"]
         if isinstance(message.extra.get("flush"), bool):
             flush = message.extra["flush"]
+        if isinstance(message.extra.get("style"), str):
```

Comment:
Maybe delete the if statement where we check if color is included as it is no longer sent.

Reply:
Well we might end up sending it in the future; I think it still makes sense to keep.

Reply:
I guess it's fine to keep. I kind of think it will be a mistake for the session to ever have an opinion about the colors the client sends after this PR though.

Reply:
I have left it there in case I missed a spot. Can remove it once we have confirmed that all is well.